submissions/mayowaShoyinka/htmlForms/contact-2.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
```

Comment:
You have a contact page already. Why are you creating another? @MayowaShoyinka

Reply:
the formal contact page was created before thats why i named this contact-2

Reply:
You're not meant to create another one.You should style the one you've created. You will need to delete this newly created file and move the changes to the older contact page created. @MayowaShoyinka

Reply:
changes made please check @ToluAkin